hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/HadoopFsGenerator.java

```diff
@@ -63,6 +63,12 @@ public class HadoopFsGenerator extends BaseFreonGenerator
       defaultValue = "4096")
   private int copyBufferSize;
 
+  @Option(names = {"-sync"},
+      description = "Type of operation to execute after a write. Supported " +
+      "options include none (default), hflush and hsync",
+      defaultValue = "none")
+  private String flushOrSync;
```

Comment:
Think Option works well with enum types, e.g. https://github.com/apache/ozone/blob/master/hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/container/ListSubcommand.java#L72-L74

Exposing `SyncOptions` as a top-level type and using it directly here looks simpler.